yarn.lock

```diff
@@ -1430,6 +1430,16 @@
     "@babel/helper-validator-identifier" "^7.16.7"
     to-fast-properties "^2.0.0"
 
+"@backstage/plugin-search-react@^1.1.0":
+  version "0.1.0"
```

Comment:
Not sure what happened here - is this a remnant of an earlier typo for a 1.x major? This block should probably just be removed.

Reply:
Yes, also saw that. Will remove it. 👍